src/displayapp/screens/Steps.cpp

```diff
@@ -28,6 +28,8 @@ Steps::Steps(Pinetime::Applications::DisplayApp* app,
 
   stepsCount = motionController.NbSteps();
   currentTripSteps = stepsCount - motionController.GetTripSteps();
+  currentTripStepsDistance = motionController.GetTripSteps() * 0.0008;
```

Comment:
This is doing conversion from an integer to a floating-point and then back, and that's not very efficient. It would be a lot better to replace the multiplication with 0.0008 with a division by 1250.
```suggestion
  currentTripStepsDistance = motionController.GetTripSteps() / 1250;
```

Reply:
Eh, good point.